packages/ag-charts-community/src/chart/toolbar/toolbarStyles.ts

```diff
@@ -2,31 +2,31 @@ import { ToolbarPosition } from './toolbarTypes';
 
 export const block = 'ag-charts-toolbar';
 export const elements = {
-    align: `${block}__align`,
-    button: `${block}__button`,
-    icon: `${block}__icon`,
-    label: `${block}__label`,
+    align: 'ag-charts-toolbar__align',
```

Comment:
I'd be inclined to keep the variables here even if we can't use them in the `.css` files themselves. Saves typos.

Reply:
I went this way so it's easier to validate the JS and CSS class names match exactly, and stuff like ctrl + F will work. Additionally too, in the future, we may be able to add tools that detect unused classes, and class names that don't point to anything in the CSS - but they only work if you spell out your classes in full without any abstraction